src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -500,14 +500,27 @@ public Task GetTimeSeriesConfig()
                     {
                         timeSeriesCollection[collection.Key] = collection.Value.ToJson();
                     }
+                    
+                    var configJson = new DynamicJsonValue
+                    {
+                        [nameof(timeSeriesConfig.Collections)] = timeSeriesCollection,
+                        [nameof(timeSeriesConfig.PolicyCheckFrequency)] = timeSeriesConfig.PolicyCheckFrequency
+                    };
+                                        
+                    if (timeSeriesConfig.NamedValues != null)
```

Comment:
Better to implement inside TimeSeriesConfiguration a `ToJson` method via IDynamicJson interface and handle all of that there.

Reply:
nice catch!